pynxos/device.py

```diff
@@ -21,7 +22,12 @@ def __init__(self, host, username, password, transport=u'http', port=None, timeo
         self.timeout = timeout
         self.verify = verify
 
-        self.rpc = RPCClient(host, username, password, transport=transport, port=port, verify=self.verify)
+        if username and password:
+            self.client = RPCClient(host, username, password, transport=transport, port=port, verify=self.verify)
```

Comment:
I understand the reason for changing the attribute name to `client`, but I also don't think it is worth the risk that it will more than likely break other people's code.